js/fileUtils.js

```diff
@@ -9,26 +31,22 @@ export function genMediaTags(file) {
   if (typeof file === "string" && !/^[a-z]+:\/\//i.test(file)) {
     file = `${location.protocol}//${location.host}${location.pathname}${file}`;
   }
-  return new Promise((resolve, reject) => {
-    require.ensure(
-      ["jsmediatags/dist/jsmediatags"],
-      require => {
-        const jsmediatags = require("jsmediatags/dist/jsmediatags");
-        try {
-          jsmediatags.read(file, { onSuccess: resolve, onError: reject });
-        } catch (e) {
-          // Possibly jsmediatags could not find a parser for this file?
-          // Nothing to do.
-          // Consider removing this after https://github.com/aadsm/jsmediatags/issues/83 is resolved.
-          reject(e);
-        }
-      },
-      () => {
-        // The dependency failed to load
-      },
-      "jsmediatags"
-    );
-  });
+  return require.ensure(
+    ["music-metadata"],
+    require => {
+      const mm = require("music-metadata");
+      return sourceToStream(file).then(stream => {
+        stream.type = stream.type ? stream.type.split(";")[0] : stream.type; // Strip off: ; charset=UTF-8
```

Comment:
Are we sure type will never be empty, and will always contain `;`?

Reply:
Empty or not containing `;` is not issue I believe, undefined or null is different story.
I will consider this to move this to music-metadata. 

An addition to that, f the type cannot be determined, I can add a fallback to use the actual data to determine the content type: Borewit/music-metadata#110